tests/unit/OffsetLedger.spec.js

```diff
@@ -1,6 +1,6 @@
 const assert = require('assert');
 
-const OffsetLedger = require('../../lib/OffsetLedger');
+const OffsetLedger = require('../../lib/OffsetLedger').default;
```

Comment:
I think this is the way, the same is done in arsenal.

Maybe this is due to the dynamic transpiling of library's TS code, in the tests...
And the proper solution would be to migrate the test to typescript (or possibly use ES6 `import` instead of `require`), but this is not in scope of this ticket: which only deals with "bootstraping", i.e. having the toolchain in place so that it is _possible_ to start writing components in TS)